draft/2022-09-14-this-week-in-rust.md

```diff
@@ -153,27 +140,28 @@ Rusty Events between 2022-09-14 - 2022-10-12 🦀
     * [**Going full stack on Rust**](https://www.meetup.com/dutch-rust-meetup/events/286727064/)
 * 2022-10-02 | Florence, IT + Virtual | [RustLab](https://rustlab.it/)
```

Comment:
May want to specify that this is lasting 2 days:

```suggestion
* 2022-10-02 | Florence, IT + Virtual | [RustLab (Oct 2-4)](https://rustlab.it/)
```

Reply:
For this case, the event itself is marked as running for two days (RustLab Conference 2022 (Oct 2-4)), so I don't think we need to mark the group itself with the same information. Good idea though to make sure it's clear to run across those days!